### DISCUSSION TITLE
Using color in log window using ANSI escape sequences

### DISCUSSION THREAD

**Discussion by sab4tg** [Vector35/binaryninja-api#2144] [Q&A] [answered]
Hello!

Wondering if colorizing text within the Log window is possible using ANSI escape sequences? So for example if I wanted red text to appear as output from some piece of analysis, I could write ```print('\033[91m' + 'some red text')```

I have seen some examples that use this within the python/examples repo (cli_dis & cli_lift are two off the top of my head), but when I clone these and try to use these features, the escape character doesn't work and I only see the following in my Log window:

![image](https://user-images.githubusercontent.com/8622706/100039537-69b8cc00-2dd3-11eb-9660-649841300304.png)

Just wanted to confirm this feature is still supported, and if there's any insight into what might be causing the output I'm seeing? Thank you very much in advance!

**Comment by psifertex** [comment, marked as answer]
This examples were meant to be run headlessly in a console if I recall correctly. The log window has never supported them. For more control over display options such as font color, I recommend using the show_*_report APIs: 

https://api.binary.ninja/search.html?q=show_+report&check_keywords=yes&area=default#
